app/scripts/modules/core/src/naming/moniker.model.ts

```diff
@@ -0,0 +1,8 @@
+
+export class Moniker {
+  application: string;
+  cluster: string;
+  detail?: string;
+  stack?: string;
+  sequence?: string;
```

Comment:
I've been representing this as an `Integer` in the backend - does the old naming service store sequence as a string as well?

Reply:
@lwander - It does expect it to be a string

```
  expect(this.namingService.getSequence('app-cluster-v003')).toBe('v003');
```